poshc2/server/Payloads.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, KillDate, Key, Insecure, UserAgent, Referrer, ConnectURL, Bas
         self.KillDate = KillDate
         self.Key = Key
         self.QuickCommand = select_item("QuickCommand", "C2Server")
+        self.FirstURL = get_first_url(select_item("PayloadCommsHost", "C2Server"), select_item("DomainFrontHeader", "C2Server"))
```

Comment:
Longer term we should split the array out when we create the C2 server (or a new payload) and add a new row in the URLs table for each one, then for this stuff you can use `DB.get_default_url_id()`.

Would then allow you use `show-urls` to see all the urls, and can set the failover etc just using IDs or names from the list.

Reply:
Yeah we can update that going forward definitely now that multiple URLs have been added